CSS/ex_17/google_replicate.html

```diff
@@ -0,0 +1,197 @@
+            margin-top: 30px;
+            margin-bottom: 20px;
+        }
+        .button-container button {
+            margin-right: 5px;
+            background: #f2f2f2;
+            color: #222;
+            border: 0;
+            border-radius: 4px;
+            padding: 8px 16px;
+        }
+        .language-container {
+            display: inline-block;
+            line-height: 28px;
+            text-align: center;
+        }
+        section {
+            display: flex;
+            flex-grow: 1;
+            flex-shrink: 0;
+            justify-content: center;
+        }
+        section p {
+            text-align: center;
+            font-family: Arial;
+            font-size: 13px;
+            color: rgb(34, 34, 34);
+        }
+        section a{
+            color: rgb(102, 0, 153);
+            font-size: 13px;
+            cursor: pointer;
+        }
+        footer {
+            display: flex;
+            flex-direction: column;
+            background-color: rgb(242, 242, 242);
+            margin-top: 40px;
+            border: 1px;
+            font-size: 14px;
+            color: rgb(34, 34, 34);
+        }
+       footer > .footer-top {
+            display: flex;
+            height: 15px;
+            padding-top: 15px;
+            padding-bottom: 15px;
+            padding-left: 30px;
+            padding-right: 15px;
+            border-bottom: solid rgb(228, 228, 228);
+            color: rgba(0, 0, 0, 0.54);
+        }
+        footer > .footer-bottom {
+            display: flex;
+            justify-content: space-between;
+            height: 15px;
+            padding-top: 15px;
+            padding-bottom: 15px;
+            padding-left: 30px;
+            padding-right: 15px;
+            border-bottom: solid rgb(228, 228, 228);
+            color: rgba(0, 0, 0, 0.54);
+        }
+
+    </style>
+</head>
+<body>
+    <nav>
+        <div>
+            <a href="#">About</a>
+            <a href="#">Store</a>
+        </div>
+        <div class="nav-right">
+            <a href="#">Gmail</a>
+            <a href="#">Images</a>
+            <img src="menu.png" class="menu-image"/>
+            <img src="http://placehold.it/50x50" class="profile-picture" />
+        </div>
+    </nav>
+
+    <main>
+        <img src="google-logo.png" alt="google-logo" class="google-logo">
+        
+        <form action="https://www.google.ca/search" method="GET">
+            <div class="search-container">
+            <img src="search-icon.png" alt="search-icon" />
+            <input type="search" name="q" />
+            <img src="voice-icon.png" alt="voice-icon" />
+            </div>
+
+            <div class="button-container">
+            <button>Google Search</button>
+            <button>I'm Feeling Lucky</button>
+            </div>
+        </form>
+    </main>
+
+    <section>
+        <div>
+        <p>Google offered in: <a href="#">Français</a></p>
+        </div>
+    </section>
```

Comment:
Would be nice to add some class to section or nested div, so they would have meaning, context. Something like 
<section class="translation"> or <section class="ofrd-lang"> And then instead styling section {} you would style .translation {}. Otherwise if you style section, if you will have to use more sections in your page, all of them will have this same styling. And maybe you will not want all of them to be same. Just something to consider.